packages/algolia/test/client.test.ts

```diff
@@ -153,6 +159,22 @@ describe('Algolia Search Client', () => {
     });
   });
 
+  test('Should search event entity', async () => {
+    reverseAlgoliaSearchIndex.search.mockResolvedValueOnce(searchEventResponse);
+
+    const response = await algoliaSearchClient.search(
+      ['event'],
+      'query',
+      {},
+      true,
+    );
+    console.log(response.hits[0]);
```

Comment:
This should be thrown by eslint. I think this stage I want the prettier commits taken out of this branch and es lint and its fixed put into the new branch to be merged separately before this